themes/cardinal_service/pdb_components/opportunities_list/src/app.jsx

```diff
@@ -34,11 +34,12 @@ ReactDOM.render(
     showMoreFilters
     useGrid
     bundle={nodeBundle}
-    mainFiltersCount={3}
+    mainFiltersCount={7}
     fields={nodeFields}
     header={<h2>Search by</h2>}
+    attributes={"flex-10-of-12"}
```

Comment:
if this is an attributes, it should be a key value object. like `wrapperAttributes={{class: 'flex-10-of-12', another-attribute: 'some value'}}`
otherwise, i like this approach